src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -287,73 +283,94 @@ There are a few options to implement Auth depending on how much customization yo
 
 <br/>
 
-**Recap:** We recommend using the Authenticator UI component for quick and easy set up and then use the Amplify Libraries to customize the user experience and logic as needed. Once you decide which option will work for your use case you can continue implementing authentication on your front-end application.
+We recommend using the Authenticator UI component for quick and easy set up and then use the Amplify Libraries to customize the user experience and logic as needed. Once you decide which option will work for your use case you can continue implementing authentication on your frontend application. If you prefer to work directly with the APIs, you can review the [*Enable Sign-up, Sign-in, and Sign-out*](https://docs.amplify.aws/lib/auth/emailpassword/q/platform/js/) guide to learn more about building with the Amplify Libraries.
```

Comment:
We recommend using the Authenticator UI component for quick and easy set up and then use the Amplify Libraries to customize the user experience and logic as needed. Once you decide which option will work for your use case you can continue implementing authentication on your frontend application. If you prefer to work directly with the APIs, you can review the [*Enable Sign-up, Sign-in, and Sign-out*](https://docs.amplify.aws/lib/auth/emailpassword/q/platform/js/) guide.

Reply:
Updated